Lib/enum.py

```diff
@@ -1083,6 +1083,13 @@ class Enum(metaclass=EnumType):
     attributes -- see the documentation for details.
     """
 
+    @classmethod
+    def __signature__(cls):
+        if cls._member_names_:
+            return '(*values)'
```

Comment:
Why `*values`? If I understand correctly this corresponds to calls like `Color(1)`, which only accept exactly one argument.

Reply:
In 3,12 multiple values are accepted to match multiple values being used to create the member:

    class Point(Enum):
        ORIGIN = 0, 0

    >>> Point(0, 0)
    <Point.ORIGIN: (0, 0))>